### PR TITLE
chore: update node to v20 and npm to v9 or newer

### PR DESCRIPTION
**Pull request by mrgoltstein**
This PR upgrades node & npm versions to latest LTS so that dependabot can resume normal operations.

This Pull Request fulfills the following requirements:

- [x] The commit message follows our guidelines.
- [x] Tests for the changes have been added if needed.
- [x] Does not affect documentation or it has been added or updated.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
   "devDependencies": {
     "@commitlint/cli": "^17.7.1",
     "@commitlint/config-conventional": "^17.7.0",
-    "@ridedott/eslint-config": "^2.23.1",
+    "@ridedott/eslint-config": "^2.23.75",
```
Comment (dicearr) [inline package.json:16]: Are this changes related to the Node bump?

Comment (mrgoltstein) [reply package.json:16]: Yeah it seems so. This rule started breaking in this PR.

Comment (dicearr) [reply package.json]: Reverted in https://github.com/ridedott/merge-me-action/pull/1819/commits/ca88ea0e068f7cc0e20557a519db9bb8d81dcec9.

**Review by griseau — APPROVED** [review]
❤️
